react/Product.d.ts

```diff
@@ -0,0 +1,50 @@
+export function parseToJsonLD({
+  product,
+  selectedItem,
+  currency,
+  decimals,
+  pricesWithTax,
+}: {
+  product: any
+  selectedItem: any
+  currency: any
+  decimals: any
+  pricesWithTax: any
+}): {
+  '@context': string
+  '@type': string
+  '@id': string
+  name: any
+  brand: {
+    '@type': string
+    name: any
+  }
+  image: any
+  description: any
+  mpn: any
+  sku: any
+  category: any
+  offers: {
+    '@type': string
+    lowPrice: any
+    highPrice: any
+    priceCurrency: any
+    offers: any
+    offerCount: any
+  }
+}
+declare const _default: import('react').MemoExoticComponent<typeof StructuredData>
+export default _default
+declare function StructuredData({
+  product,
+  selectedItem,
+}: {
+  product: any
+  selectedItem: any
+}): JSX.Element
+declare namespace StructuredData {
+  namespace propTypes {
+    const product: any
+    const selectedItem: any
+  }
+}
```

Comment:
What do you think about adding a blank line between the declarations? I think it will improve readability.